execution/executor/src/tests/mod.rs

```diff
@@ -24,7 +24,6 @@ use aptos_types::{
     block_info::BlockInfo,
     chain_id::ChainId,
     ledger_info::{LedgerInfo, LedgerInfoWithSignatures},
-    proof::definition::LeafCount,
```

Comment:
Any particular reason the imports were removed and fully-qualified names used?

Reply:
Fixed it. I was feature gating the test before, but ended up using cfg_attr with ignore because it was neater.